File: hw/collahuazo/u3/hw23AbstractFactory/AbstractFactoryJava/Button/src/main/java/ec/edu/espe/abstractfactory/controller/LinuxFactory.java
package ec.edu.espe.abstractfactory.controller;
/**
 *
 * @author Brandon Collahuazo,CodeBros,@ESPE
 */
import ec.edu.espe.abstractfactory.model.*;

public class LinuxFactory extends GUIFactory {

    public Button createButton() {
        return new LinuxButton();
    }

    public Menu createMenu() {
        return new LinuxMenu();
    }
}

File: hw/collahuazo/u3/hw23AbstractFactory/AbstractFactoryJavaScritp/controller/linuxFactory.js
import GUIFactory from './guiFactory.js';
import LinuxButton from '../model/linuxButton.js';
import LinuxMenu from '../model/linuxMenu.js';

class LinuxFactory extends GUIFactory {
  createButton() {
    return new LinuxButton();
  }

  createMenu() {
    return new LinuxMenu();
  }
}

export default LinuxFactory;
